site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -2373,4 +2373,27 @@ private function getStats(Gradeable $gradeable, User $grader, bool $full_stats,
             $total_total += $value * $num_components;
         }
     }
+    
+    /**
+     * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/feedback/set", methods={"POST"})
+     */
+    public function ajaxSetPeerFeedback($gradeable_id) {
```

Comment:
Can we add access control here to make certain we are allowed to update this feedback?

Reply:
The blank return statemetns in the function should act as access control: ending the function early if the user doesn't have permission to update feedback